README.md

```diff
@@ -8,88 +8,121 @@ Official [Buildkite Test Analytics](https://buildkite.com/test-analytics) collec
 
-    ```bash
-    # If you use npm:
-    npm install --save-dev buildkite-test-collector
+   ```bash
+   # If you use npm:
+   npm install --save-dev buildkite-test-collector
 
-    # or, if you use yarn:
-    yarn add --dev buildkite-test-collector
-    ```
+   # or, if you use yarn:
+   yarn add --dev buildkite-test-collector
+   ```
 
-3) Add the buildkite test collector to your testing framework:
+3. Add the buildkite test collector to your testing framework:
 
-    ### Jest
+   ### Jest
 
-    Update your [Jest configuration](https://jestjs.io/docs/configuration):<br>
+   Update your [Jest configuration](https://jestjs.io/docs/configuration):<br>
 
-    ```js
-      // jest.config.js
+   ```js
+     // jest.config.js
 
-      // Send results to Test Analytics
-      reporters: [
-        'default',
-        'buildkite-test-collector/jest/reporter'
-      ],
+     // Send results to Test Analytics
+     reporters: [
+       'default',
+       'buildkite-test-collector/jest/reporter'
+     ],
 
-      // Enable column + line capture for Test Analytics
-      testLocationInResults: true
-    ```
+     // Enable column + line capture for Test Analytics
+     testLocationInResults: true
+   ```
 
-    ### Jasmine
+   If you would like to pass in the API token using a custom environment variable, you can do so using the report options.
 
-    [Add the buildkite reporter to jasmine](https://jasmine.github.io/setup/nodejs.html#reporters):<br>
+   ```js
+   // jest.config.js
 
-    ```js
-      // SpecHelper.js
-      var BuildkiteReporter = require('buildkite-test-collector/jasmine/reporter');
-      var buildkiteReporter = new BuildkiteReporter();
+   // Send results to Test Analytics
+   reporters: [
+     "default",
+     ["buildkite-test-collector/jest/reporter", { token: INSERT_TOKEN_HERE }],
+   ];
+   ```
 
-      jasmine.getEnv().addReporter(buildkiteReporter);
-    ```
+   ### Jasmine
 
-    ### Mocha
+   [Add the buildkite reporter to jasmine](https://jasmine.github.io/setup/nodejs.html#reporters):<br>
 
-    [Install mocha-multi-reporters](https://github.com/stanleyhlng/mocha-multi-reporters) in your project:<br>
+   ```js
+   // SpecHelper.js
+   var BuildkiteReporter = require("buildkite-test-collector/jasmine/reporter");
+   var buildkiteReporter = new BuildkiteReporter();
 
-    ```
-      npm install mocha-multi-reporters --save-dev
-    ```
+   jasmine.getEnv().addReporter(buildkiteReporter);
+   ```
 
-    and configure it to run your desired reporter and the Buildkite reporter
+   If you would like to pass in the API token using a custom environment variable, you can do so using the report options.
 
-    ```js
-      // config.json
-      {
-        "reporterEnabled": "spec, buildkite-test-collector/mocha/reporter"
-      }
-    ```
+   ```js
+   // SpecHelper.js
+   var buildkiteReporter = new BuildkiteReporter(undefined, {
```

Comment:
`undefined` looks a little awkward here. Is there anything we could in JS to improve this method signature?
No worries if there isn't a quick fix.

Reply:
Hmm not that I know of, since we're defaulting it later to `{ cwd: process.cwd() }`.
Can always change the instructions to explicitly set the cwd? Does this look better?

```js
var buildkiteReporter = new BuildkiteReporter({ cwd: process.cwd() }, {
     token: INSERT_TOKEN_HERE,
   });
```

Reply:
I don't have strong feelings about it, I personally prefer the `undefined` version over having to set the `cwd`. So maybe leave it as is?